buildSrc/src/main/java/quilt/internal/decompile/quiltflower/QuiltflowerDecompiler.java

```diff
@@ -37,6 +38,10 @@ public QuiltflowerDecompiler(Project project) {
     public void decompile(File file, File outputDir, Map<String, Object> options, Collection<File> libraries) {
         Path outputPath = outputDir.toPath();
 
+        // disable "inconsistent inner class" warning due to spam in the logs
+        options.put(IFernflowerPreferences.WARN_INCONSISTENT_INNER_CLASSES, "0");
+        // note: "inconsistent generic signature" warnings should also be suppressed, but that isn't currently option in quiltflower
```

Comment:
We could suppress these by filtering them out in `LoggerImpl#writeMessage` below

Reply:
true, but I think it would be better to not bandaid it and add an option to quiltflower
we're using bleeding-edge quiltflower anyway!